content/id/docs/tasks/run-application/horizontal-pod-autoscale.md

```diff
@@ -0,0 +1,459 @@
+---
+reviewers:
+- phanama
+- wahyuoi
+title: Horizontal Pod Autoscaler
+feature:
+  title: Horizontal scaling
+  description: >
+    Scale your application up and down with a simple command, with a UI, or automatically based on CPU usage.
+
+content_template: templates/concept
+weight: 90
+---
+
+{{% capture overview %}}
+
+Horizontal Pod Autoscaler secara otomatis akan men-*scale* jumlah *pod* didalam kontroler replikasi, *deployment*, 
+*replica set* ataupun *stateful* berdasarkan hasil observasi penggunaan CPU(atau, dengan 
+[*costum* metrik](https://git.k8s.io/community/contributors/design-proposals/instrumentation/custom-metrics-api.md), pada beberapa aplikasi yang menyediakan metrik). 
+Perlu dicatat bahwa Horizontal Pod Autoscaling tidak dapat diterapkan pada objek yang tidak dapat di-*scale*, seperti DeamonSets. 
+
+Horizontal Pod Autoscaler diimplementasikan sebagai Kubernetes API *resource* dan sebuah kontroller.
+*Resource* tersebut akan menentukan perilaku dari kontrolernya.
+Kontroler akan menyesuaikan jumlah replika pada kontroler replikasi atau pada *deployment* untuk menyesuaikan dengan hasil observasi rata-rata
+penggunaan CPU sesuai dengan yang ditentukan oleh pengguna.
+
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+## Bagaimana cara kerja Horizontal Pod Autoscaler?
+
+![Horizontal Pod Autoscaler diagram](/images/docs/horizontal-pod-autoscaler.svg)
+
+Horizontal Pod Autoscaler diimplementasikan sebagai sebuah *control loop*, yang secara
+berkala dikontrol oleh *flag* `--horizontal-pod-autoscaler-sync-period` pada kontroler manajer
+(dengan nilai standar yaitu 15 detik). 
+
+Dalam setiap periode, kontroler manajer melakukan kueri penggunaan sumber daya dan membandingkan
+dengan metrik yang dispesifikasikan pada HorizontalPodAutoscaler. Kontroler manajer mendapat
+metrik dari *resource* metrik API (untuk metrik per *pod*) atau dari *custom* metrik API (untuk semua metrik lainnya).
+
+During each period, the controller manager queries the resource utilization against the
+metrics specified in each HorizontalPodAutoscaler definition.  The controller manager
+obtains the metrics from either the resource metrics API (for per-pod resource metrics),
+or the custom metrics API (for all other metrics).
+
+* Untuk metrik per *pod* (seperti CPU), kontroler mengambil metrik dari *resource* metrik API
+  untuk setiap *pod* yang ditargetkan oleh HorizontalPodAutoscaler. Kemudian, jika nilai target penggunaan ditentukan,
+  maka kontroler akan menghitung nilai penggunaan sebagai persentasi dari pengguaan *resource* dari kontainer 
```

Comment:
Container is also Kubernetes API object https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#container-v1-core
```suggestion
  maka _controller_ akan menghitung nilai penggunaan sebagai persentasi dari pengguaan sumber daya dari Container 
```